base/commands/config/config_add.go

```diff
@@ -0,0 +1,54 @@
+//go:build base
+
+package config
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"path/filepath"
+	"strings"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	"github.com/hazelcast/hazelcast-commandline-client/clc/config"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+)
+
+type AddCmd struct{}
+
+func (cm AddCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("add TARGET")
+	short := "Adds a configuration"
+	long := "Adds a configuration with the given name/path and KEY=VALUE pairs"
+	cc.SetCommandHelp(long, short)
+	cc.SetPositionalArgCount(1, math.MaxInt)
+	return nil
+}
+
+func (cm AddCmd) Exec(ctx context.Context, ec plug.ExecContext) error {
```

Comment:
ctx is unused parameter. Can we use _ for this?

Reply:
Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/171/commits/4f3ef69e198f6a0a45ff80a90627c33291f72444